e2e/svelte/scripts/setupTypeScript.js

```diff
@@ -3,99 +3,104 @@
 /** This script modifies the project to support TS code in .svelte files like:
 
   <script lang="ts">
-    export let name: string;
+  	export let name: string;
   </script>
-
+ 
   As well as validating the code for CI.
   */
 
 /**  To work on this script:
   rm -rf test-template template && git clone sveltejs/template test-template && node scripts/setupTypeScript.js test-template
 */
 
-const fs = require('fs')
-const path = require('path')
-const { argv } = require('process')
+import fs from "fs"
```

Comment:
why convert everything to double quotes? that's not too common in the JS ecosystem. it might be more standard to format with single quotes

Reply:
I copied the file from the svelte template and did no formatting. `e2e` has been neglected so far in terms of common formatting :)